subprojects/docs/src/main/resources/header.html

```diff
@@ -234,6 +219,13 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                 </ul>
             </li>
         </ul>
+        <h3 id="appendix">Appendix</h3>
```

Comment:
Hmm.. I feel that the need for an Appendix highlights that we're lacking organization. However, I think we should be careful not to put stuff under the heading it really belongs just because that list looks too long. I think we should first aim for consistency before we optimize how it's split up, if that makes sense. 

BTW, I'm okay with a future structure such that not everything ends up in the side nav. A bunch of stuff could be split or merged into other chapters. Or gathered onto a page like you did with "Core plugins".

Reply:
I think it's wrong to consider an appendix a failure of organisation. It's a valid and useful part of books. It mostly consists of reference-style material right now, which given some thought could be added to the Reference section of the nav without bloating it. Not sure.

Reply:
Okay, I agree it can work, but I guess maybe it's unclear to me how the appendix differs from reference.